CHANGELOG.md

```diff
@@ -7,7 +7,23 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [7.0.12] - 2023-11-16
 
-- Adds Phone Number normalisation
+In this release, the core API routes have been updated to incorporate phone number normalization before processing. Consequently, existing entries in the database also need to undergo normalization. To facilitate this, we have included a migration script to normalize phone numbers for all the existing entries.
+
+**NOTE**: You can skip the migration if you are not using passwordless via phone number.
+
+### Migration steps
+
```

Comment:
Write what this migration script is doing. Something like "It normalises existing phone numbers in the database if they are unnormalised"

Reply:
done